### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by varshan-ms**

### CODE REVIEW

**Comment by varshan-ms** [comment]
First version of grid

File: src/grid.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+.gd-cols-eq { @include gd;
+              grid-template-columns: repeat(var(--gd-cols-eq, 12), 1fr);
+}
+.gd-cols-cus { @include gd;
+               grid-template-columns: var(--gd-cols-cus, repeat(12, 1fr));
+}
+.gd-col-subgrid { @include gd;
+                  grid-template-columns: subgrid;
+                  grid-column: var(--gd-col-cus, repeat(12, 1fr));
```
Comment (babu-shanmugam) [inline src/grid.scss:68]: I think the default value for this is not right. This default value will be good for grid-template-columns. Instead we should have `span 1` as the default

File: src/grid.scss [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+ *
+ *                    subgrid (subgrid within col)
+ *
+ *                      Variables used:
+ *                        [Modifier 1: col]
+ *                          --gd-col-cus ( for cus subgrid within a col )
+ *
+ */
+
+.gd-gap-cus-m { gap: $grid-m; }
```
Comment (babu-shanmugam) [inline src/grid.scss:46]: SCSS variable can be named in this format. $gd-gap-m would have been good in this case. Consider renaming all the other variables in this same way.

File: src/grid.scss [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+ *                          --gd-cols-cus-l   (num of cus cols @l )
+ *                          --gd-cols-cus-xl  (num of cus cols @xl )
+ *                          --gd-cols-cus-2xl (num of cus cols @2xl )
+ *
+ *                        [Modifier 1: gap]
+ *                          --gd-gap-cus     ( gap size s )
+ *                          --gd-gap-cus-m   ( gap size m )
+ *                          --gd-gap-cus-l   ( gap size l )
+ *                          --gd-gap-cus-xl  ( gap size xl )
+ *                          --gd-gap-cus-2xl ( gap size 2xl )
```
Comment (babu-shanmugam) [inline src/grid.scss:36]: This is a different modifier but is part of cus.

**Pull request closed without merging**